dwave/cloud/exceptions.py

```diff
@@ -88,3 +75,9 @@ class ProblemStructureError(InvalidProblemError):   # inherit for backwards comp
 
 class ProblemUploadError(Exception):
     """Problem multipart upload failed."""
+
+
+# for backward compatibility
+from dwave.cloud.api.exceptions import RequestTimeout
+
+from dwave.cloud.config.exceptions import *
```

Comment:
Also planned for deprecation at some point?

Reply:
Not any time soon, I'd say.